src/proc.cpp

```diff
@@ -965,13 +965,17 @@ int Proc::read_system() //
         else
             flag_thread_ok = false;
 
+#ifdef LINUX
+				//Ubuntu does not updates 'shedstat' if first thread is sleep and another threads are works
+				flag_schedstat = false;
+#else				
```

Comment:
Not sure for 100% but it's kernel feature. That's mean it's very weakly depend of distro.